addons/mail/models/mail_channel.py

```diff
@@ -1317,24 +1317,26 @@ def channel_create(self, name, privacy='public'):
         return channel_info
 
     @api.model
-    def group_chat_create(self, partners_to):
+    def group_chat_create(self, current_partner, partners_to):
         """ Create a private group chat channel.
+            :param current_partner : res.partner id who created the conversation
             :param partners_to : list of res.partner ids to add to the conversation
             :returns: channel_info of the created or existing channel
             :rtype: dict
         """
-        if self.env.user.partner_id.id not in partners_to:
-            partners_to.append(self.env.user.partner_id.id)
+        if current_partner not in partners_to:
+            partners_to.append(current_partner)
```

Comment:
not needed, just remove this code and param, and make sure `partners_to` contains all partners